#!/usr/bin/python3

def raise_exception_msg(message=""):
    """fnc to raise name error with message"""
    raise NameError(message)
